service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/CommentType.java

```diff
@@ -0,0 +1,14 @@
+package uk.nhs.adaptors.gp2gp.ehr.mapper;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public enum CommentType {
+    AGGREGATE_COMMENT_SET("AGGREGATE COMMENT SET"),
+    LABORATORY_RESULT_COMMENT("LABORATORY RESULT COMMENT(E141)"),
```

Comment:
in excel in row 154 there is a comment type LABORATORY RESULT DETAIL(E136) that should be populated if the value is provided

Reply:
👍  Missing mappings have been added to the Observation mapper